test/fixtures/Test-non-existing-content_3508203637/Getting-XDM-README-from-wrong-URL-_3748211873/recording.har

```diff
@@ -8,130 +8,139 @@
+              "value": "application/vnd.github.VERSION.raw\""
+            },
+            {
+              "name": "user-agent",
+              "value": "project-helix"
+            },
             {
               "name": "host",
-              "value": "raw.githubusercontent.com"
+              "value": "api.github.com"
             }
           ],
-          "headersSize": 111,
+          "headersSize": 180,
           "httpVersion": "HTTP/1.1",
           "method": "GET",
-          "queryString": [],
-          "url": "https://raw.githubusercontent.com/nobody/xdm/master/README.md"
+          "queryString": [
+            {
+              "name": "ref",
+              "value": "master"
+            }
+          ],
+          "url": "https://api.github.com/repos/nobody/xdm/contents/README.md?ref=master"
         },
         "response": {
-          "bodySize": 15,
+          "bodySize": 106,
           "content": {
-            "mimeType": "text/plain",
-            "size": 15,
-            "text": "404: Not Found\n"
+            "mimeType": "application/json; charset=utf-8",
+            "size": 106,
+            "text": "{\"message\":\"Not Found\",\"documentation_url\":\"https://developer.github.com/v3/repos/contents/#get-contents\"}"
           },
           "cookies": [],
           "headers": [
             {
-              "name": "content-security-policy",
-              "value": "default-src 'none'; style-src 'unsafe-inline'; sandbox"
-            },
-            {
-              "name": "strict-transport-security",
-              "value": "max-age=31536000"
+              "name": "server",
+              "value": "GitHub.com"
             },
             {
-              "name": "x-content-type-options",
-              "value": "nosniff"
+              "name": "date",
+              "value": "Mon, 15 Apr 2019 11:35:00 GMT"
             },
             {
-              "name": "x-frame-options",
-              "value": "deny"
+              "name": "content-type",
+              "value": "application/json; charset=utf-8"
             },
             {
-              "name": "x-xss-protection",
-              "value": "1; mode=block"
+              "name": "content-length",
+              "value": "106"
             },
             {
-              "name": "x-github-request-id",
-              "value": "97F8:6405:1EE70B:2131A6:5C7528E7"
+              "name": "connection",
+              "value": "close"
             },
             {
-              "name": "content-length",
-              "value": "15"
+              "name": "status",
+              "value": "404 Not Found"
             },
             {
-              "name": "accept-ranges",
-              "value": "bytes"
+              "name": "x-ratelimit-limit",
+              "value": "60"
             },
             {
-              "name": "date",
-              "value": "Tue, 26 Feb 2019 11:54:16 GMT"
+              "name": "x-ratelimit-remaining",
```

Comment:
60 req/minute?
Können wir upgraden? Ansonsten weiß ich nicht wie wir weiter vorgehen sollen. GHs caching auf .raw zerschießt und einmal komplett die cache invalidation und das ist so wie ich das sehe inakzeptabel…

Reply:
https://developer.github.com/v3/#rate-limiting

> For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the user making requests.

Reply:
If we manage to pass in a token, we can raise the limit to 5000 requests per hour (per user).

Reply:
I guess the 5k limit is also per user per gh application? Or per user across al gh applications? I mean the goal is to have it all work from the helix-bot, so we might just generate a token for the user's gh bot installation…
Well, anyways I guess this will linger here for a while until I get the helix bot to a more advanced stage…

Reply:
> This means that all OAuth applications authorized by a user share the same quota of 5000 requests per hour when they authenticate with different tokens owned by the same user.